artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/ActiveMQConnection.java

```diff
@@ -673,12 +673,16 @@ protected final void checkClosed() throws JMSException {
       }
    }
 
-   public void authorize() throws JMSException {
+   public void authorize(boolean validateClientId) throws JMSException {
```

Comment:
public interface change in client code, should probably keep the existing method for people who may have coded against it.

Reply:
@michaelandrepearce that's internal API though

Reply:
The method was public I will add it back.